sample-app/src/main/java/kin/devplatform/sample/MainActivity.java

```diff
@@ -125,8 +127,31 @@ public void onClick(View v) {
 			}
 		});
 
-		addNativeSpendOffer(nativeSpendOffer);
-		addNativeOfferClickedObserver();
+		startSdk();
+	}
+
+	private void startSdk() {
+		/**
+		 * SignInData should be created with registration JWT {see https://jwt.io/} created securely by server side
+		 * In the the this example {@link SignInRepo#getJWT} generate the JWT locally.
+		 * DO NOT!!!! use this approach in your real app.
+		 * */
+		String jwt = SignInRepo.getJWT(this);
+		Kin.start(getApplicationContext(), jwt, Environment.getPlayground(), new KinCallback<Void>() {
```

Comment:
I don't see any problem but just want to understand why you decided to move this from the App to the this activity

Reply:
The reason is that:
* Now `start` shouldn't be called specifically from app, as we know to recover from process restart.
* For testing sample app, we must have it here, both for simulating process restart without init sdk on `Application.onCreate()`. and for testing that we're blocking each API before start is finished.